sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -830,6 +830,10 @@ def test_neighbors_badargs():
         assert_raises(ValueError,
                       nbrs.predict,
                       [])
+        if isinstance(cls, neighbors.KNeighborsClassifier) or \
+                isinstance(cls, neighbors.KNeighborsRegressor):
```

Comment:
again style: please use enclosing parens for the if condition instead of the `\`-escaping of the new line.